.spellignore

```diff
@@ -565,3 +565,17 @@ USD
 EUR
 CCO
 untagged
+expiryDate
+licenseDuration
+licenseId
+licenseName
+licenseStatus
+licenseType
+Visio
+Orgs
+Org
+RightScale
+MCA
+IOT
```

Comment:
Should be IoT rather than IOT I would have thought

Reply:
This appears to be based on a category that appears in our platform as IOT